spacegrep/src/lib/Semgrep.ml

```diff
@@ -51,7 +51,7 @@ let make_semgrep_json doc_matches : Semgrep_t.match_results =
           let ((pos1, _), (_, pos2)) = match_.region in
           let metavars = List.map convert_capture match_.named_captures in
           let lines =
-            Src_file.region_of_pos_range src pos1 pos2
+            Src_file.lines_of_pos_range src pos1 pos2
             |> String.split_on_char '\n'
           in
           let extra = {
```

Comment:
This change results in returning the full line(s) containing the match rather than just the match:
```
function() {
  hello("world");
}
```
If `"hello"` matches, json `lines` field is now populated with `["  hello(\"world\");"; ""]`.

Reply:
So why the CI didn't fail on this PR? Why the error came later?